pwiz_tools/Skyline/FileUI/FileUIResources.resx

```diff
@@ -658,4 +658,7 @@ Would you like to remove some or all of this suffix to shorten the names used in
   <data name="ChooseIrtStandardPeptidesDlg_ChooseIrtStandardPeptidesDlg_Unavailable__no_matching_iRT_standards_found" xml:space="preserve">
     <value>Unavailable: no matching iRT standards found</value>
   </data>
+  <data name="OpenFileDialogNE_OpenFileDialogNE_Open" xml:space="preserve">
+	  <value>Open</value>
```

Comment:
This one's still a tab. :)

Reply:
I didn't add that one.
Resharper added that one.
Nick showed me to add the string constant in the code and then hit F6 and resharper added the string to the resx and put the resx lookup in the code.
That is where this tab with spaces came from.

I guess my Visual Studio settings need updating.